tasks/configure.js

```diff
@@ -152,6 +152,11 @@ async function task (opts) {
 		}
 	});
 
+	const invalidVariables = Object.keys(patch).filter(prop => patch[prop] === '' || typeof patch[prop] === 'number');
```

Comment:
If it is possible to add a test for this that'd be great, but I'm not sure if this repository has tests.

Reply:
there is a test for this task (https://github.com/Financial-Times/n-heroku-tools/blob/225fc1bac437dce61928f41b60b06291edf50a4b/tasks/configure.test.js), adding this would be nice but currently it looks like it's not covering any error handling at all so it might be too much effort to be worth it